scripts/git_clean.sh

```diff
@@ -1,15 +1,20 @@
 #!/bin/bash
 if [[ `git checkout master` ]]; then 
   for BRANCH in `git branch | grep -v "*"`; do
-    if [[ `git diff master..$BRANCH` ]]; then
-      echo "- $BRANCH needs to be fast-forward";
-      git checkout $BRANCH;
-      if [[ `git merge master --ff-only` ]]; then
+    git checkout $BRANCH > /dev/null 2>&1;
+    if [[ `git merge master --no-edit 2> /dev/null` ]]; then
+      echo "> $BRANCH merged with master";
+      if [[ `git diff master` ]]; then
+	echo "> $BRANCH is different than master";
+        git reset --hard > /dev/null;
```

Comment:
if there are some differences, we make sure we clean any unmerged changes and we move on to the next branch